### DISCUSSION TITLE
to_networkx for heterograph?

### DISCUSSION THREAD

**Discussion by Jeriousman** [pyg-team/pytorch_geometric#4151] [Q&A]
I wanted to convert PyG graph into networkx one with to networkx function but it seems to only work for homogeneous graphs. Is it true? If it is, would you add to_networkx for hetero in near future?

**Comment by rusty1s** [comment]
AFAIK, `to_network` does not differentiate different node types inside each graph, so it is hard to allow `to_network` to return a heterogeneous graph in the PyG format. Any suggestions/references highly appreciated though!

**Reply by Jeriousman** [reply]
Right. I should go deeper in NetworkX to see.
